client/src/app/shared/video/video-miniature.component.html

```diff
@@ -17,7 +17,7 @@
       </a>
 
       <span class="video-miniature-created-at-views">
-        <ng-container *ngIf="displayOptions.date">{{ video.publishedAt | myFromNow }}</ng-container>
+        <ng-container *ngIf="displayOptions.date"><my-date-toggle [date]="video.publishedAt"></my-date-toggle></ng-container>
```

Comment:
Then you don't need the `ng-container`, just put the `ngIf` on the `my-date-toggle` element